src/main/java/nextstep/favorite/application/dto/GithubProfileResponse.java

```diff
@@ -0,0 +1,23 @@
+package nextstep.favorite.application.dto;
+
+public class GithubProfileResponse {
+
+    private String email;
+    private Integer age;
```

Comment:
NPE가 발생할 수 있는 wrapper 타입 대신 primitive 타입을 사용하면 어떨까요?
```suggestion
    private int age;
```